PIL/Image.py

```diff
@@ -1092,12 +1154,14 @@ def draft(self, mode, size):
         pass
```

Comment:
Re the type, I'd imagine this is `(Mode, Size)` based on the type aliases, ie. `(Text, Tuple[int, int])`; or could the tuple be bigger?